stratum/templates/podsecuritypolicy-stratum.yaml

```diff
@@ -0,0 +1,48 @@
+{{- if .Values.podsecuritypolicy.enabled }}
+apiVersion: policy/v1beta1
+kind: PodSecurityPolicy
+metadata:
+    name: 1-stratum
+spec:
+  privileged: true
+  allowPrivilegeEscalation: true
+  volumes:
+    - 'configMap'
+    - 'emptyDir'
+    - 'projected'
+    - 'secret'
+    - 'downwardAPI'
+    - 'csi'
+    - 'persistentVolumeClaim'
+    - 'ephemeral'
+    - 'hostPath'
+  allowedHostPaths:
+    - pathPrefix: "/dev"
+    - pathPrefix: "/sys"
+    - pathPrefix: "/var/log"
+    - pathPrefix: "/lib/modules/4.14.49-OpenNetworkLinux"
+    - pathPrefix: "/etc/onl"
+    - pathPrefix: "/lib/x86_64-linux-gnu/libonlp-platform-defaults.so"
+    - pathPrefix: "/lib/x86_64-linux-gnu/libonlp-platform.so"
+    - pathPrefix: "/lib/x86_64-linux-gnu/libonlp.so"
+    - pathPrefix: "/config"
+    - pathPrefix: "/lib/platform-config"
+  hostNetwork: true
+  runAsUser:
+    rule: 'RunAsAny'
+  seLinux:
+    # This policy assumes the nodes are using AppArmor rather than SELinux.
+    rule: 'RunAsAny'
+  supplementalGroups:
+    rule: 'MustRunAs'
+    ranges:
+      # Forbid adding the root group.
+      - min: 1
+        max: 65535
+  fsGroup:
+    rule: 'MustRunAs'
+    ranges:
+      # Forbid adding the root group.
+      - min: 1
+        max: 65535
```

Comment:
All these are required?

Reply:
The fields are required to be in the psp. If set the selinux to MustRunAs, it requires seLinuxOptions to be configured. So I set it to RunAsAny. The other two fields also are required, I just exclude the root group.